terragrunt/aws/glue/crawlers.tf

```diff
@@ -97,20 +95,6 @@ resource "aws_glue_crawler" "bes_crm_salesforce_production" {
     path = "s3://${var.raw_bucket_name}/bes/crm/salesforce"
   }
 
-  configuration = jsonencode(
-    {
-      CrawlerOutput = {
-        Tables = {
-          TableThreshold = 10
-        }
-      }
-      Grouping = {
-        TableGroupingPolicy = "CombineCompatibleSchemas"
-      }
-      CreatePartitionIndex = true
-      Version              = 1
-  })
-
```

Comment:
Could you try it just like so to see if it's the `CombineCompatibleSchemas` giving us grief:
```hcl
  configuration = jsonencode(
    {
      CrawlerOutput = {
        Tables = {
          TableThreshold = 10
        }
      }
      CreatePartitionIndex = true
      Version              = 1
  })
  ```